chore: Corrected README (#179)
fauna/fauna-go: README.md

## Event Streaming

The driver supports [Event Streaming](https://docs.fauna.com/fauna/current/reference/cdc/).

### Start a stream

> To get a stream token, append

Reviewer: ```suggestion
An Event Stream lets you consume events from an [event
source](https://docs.fauna.com/fauna/current/learn/cdc/#create-an-event-source)
as a real-time subscription.

To get an event source, append
```

[`eventSource()`](https://docs.fauna.com/fauna/current/reference/fql-api/schema-entities/set/eventsource/)

> or [`changesOn()`](https://docs.fauna.com/fauna/current/reference/fql-api/schema-entities/set/changeson/)

Reviewer: ```suggestion
or [`eventsOn()`](https://docs.fauna.com/fauna/current/reference/reference/schema_entities/set/eventsource)
```

> to a set from a [supported source](https://docs.fauna.com/fauna/current/reference/streaming_reference/#supported-sources).

Reviewer: ```suggestion
to a [supported Set](https://docs.fauna.com/fauna/current/reference/cdc/#sets).
```

> To start and subscribe to the stream, pass a query that produces a stream source to `StreamFromQuery()`:

Reviewer: ```suggestion
To start and subscribe to the stream, pass a query that produces an event source to `StreamFromQuery()`:
```

}
```

> In query results, the driver represents an eventSource as `fauna.EventSource` values.

Reviewer: ```suggestion
In query results, the driver represents an event source as a `fauna.EventSource` value.
```

> To start a stream from a query result, call `Stream()` and pass the `fauna.EventSource`.

Reviewer: ```suggestion
To start a stream from a query result, call `Stream()` and pass the `fauna.EventSource`.
This lets you output a stream alongside normal query results:
```

}
```

### Stream options

The [client configuration](#client-configuration) sets default query options for

> `Stream()`. To override these options, see [query options](#query-options).

Reviewer: ```suggestion
`StreamFromQuery()` and `Stream()`. To override these options, see [query options](#query-options).
```

> Use `StreamFromQuery()` to set `fauna.StartTime` and `fauna.EventCursor`
> stream options:

Reviewer: ```suggestion
The `StreamFromQuery()` and `Stream()` methods accept
[StreamOptFn](https://pkg.go.dev/github.com/fauna/fauna-go/v2#StreamOptFn)
functions as arguments.
```

> Use `fauna.StartTime` to restart a stream at a specific timestamp.

Reviewer: ```suggestion
Use `fauna.StartTime()` to restart a stream at a specific timestamp:
```

```go
streamQuery, _ := fauna.FQL(`Product.all().eventSource()`, nil)
client.StreamFromQuery(streamQuery, fauna.StreamOptFn{
    fauna.StartTime(1710968002310000),
})
```

> Use `fauna.EventCursor` to resume a stream after a disconnect:

Reviewer: ```suggestion
Use `fauna.EventCursor()` to resume a stream from an event cursor after a disconnect:
```

```go
client.StreamFromQuery(streamQuery, fauna.StreamOptFn{
    fauna.EventCursor("<cursor>"),
})
```

> | Function | Description |
> | -------- | ----------- |
> | `fauna.StartTime`  | Sets the stream start time. Accepts an `int64` representing the start time in microseconds since the Unix epoch.<br><br>The start time must be later than the creation time of the stream token. The period between the stream restart and the start time argument can't exceed the `history_days` value for source set's collection. If a collection's `history_days` is `0` or unset, the period can't exceed 15 minutes. |
> | `fauna.EventCursor`  | Resumes the stream after the given event cursor. Accepts a `string` representation of the cursor retrieved from a `fauna.Event`. |

Reviewer: ```suggestion
For supported functions, see
[StreamOptFn](https://pkg.go.dev/github.com/fauna/fauna-go/v2#StreamOptFn) in
the API reference.
```
